syntax = "proto3";

package BeeGees;

message BaseMessage {
    int32 Type = 1;
    bytes Blob = 2;    // encoded Protobuf message
}